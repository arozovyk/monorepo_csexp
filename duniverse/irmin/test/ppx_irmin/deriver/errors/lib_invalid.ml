type t = unit [@@deriving irmin { lib = "foo" }] (* should be [Some "foo"] *)
