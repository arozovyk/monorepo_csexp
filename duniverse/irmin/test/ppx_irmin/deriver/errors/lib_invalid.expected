File "lib_invalid.ml", line 1, characters 40-45:
Error: Could not process `lib' argument: must be either `Some "Lib"' or `None'
